Tiltfile

```diff
@@ -190,17 +190,16 @@ def enable_provider(name):
         ],
     )
 
-    # Apply the kustomized yaml for this provider
-    yaml = str(kustomize_with_envsubst(context + "/config"))
+    # Copy all the substitutions from the user's tilt-settings.json into the environment. Otherwise, the substitutions
+    # are not available and their placeholders will be replaced with the empty string when we call kustomize +
+    # envsubst below.
     substitutions = settings.get("kustomize_substitutions", {})
     for substitution in substitutions:
-        value = substitutions[substitution]
-        yaml = yaml.replace("${" + substitution + "}", value)
+        os.environ[substitution] = substitutions[substitution]
```

Comment:
can use os.environ.update(substitutions) to simplify a bit

Reply:
Thanks, I was looking at Tilt's docs on the Starlark syntax and didn't see that. Will make the change.